nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -520,7 +520,13 @@ function AddSubscriberDetails(props: DialogProps) {
   );
   const saveSubscribers = async () => {
```

Comment:
What happens if a csv is uploaded with some new subscribers, and some edited subscribers?

Reply:
It will reject the already existing subscribers since this is an add dialog, and it will set error for that specific subscriber because it first loops and validates the subscriber info. Also, the endpoint API that we are calling here would also return error, if we are trying to edit an existing subscriber. So, I made sure to just set the error for an edited subscriber and not add it to the list of subscribers that will be sent to the endpoint to make sure that the whole operation doesn't fail.